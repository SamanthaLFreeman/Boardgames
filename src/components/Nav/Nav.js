import React from 'react';
import { Link } from 'react-router-dom';

export const Nav = () => {
  return (
    <nav className="nav">
      <h1>Deck Building</h1>
      <div className="nav-btns">
        <Link to='/favorites'><button className="nav-btn">View Favorites</button></Link>
        <Link to='/owned'><button className="nav-btn">Owned Collection</button></Link>
        <Link to='/searchform'><button className="nav-btn">Search</button></Link>
      </div>
    </nav>
  )
}

export default Nav;