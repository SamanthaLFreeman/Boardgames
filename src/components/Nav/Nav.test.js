import React from 'react';
import { shallow } from 'enzyme';
import { Nav } from './Nav';

describe('Nav', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Nav />)
  });

  it('should match the snapshot with the data passed through', () => {
    expect(wrapper).toMatchSnapshot();
  });
})