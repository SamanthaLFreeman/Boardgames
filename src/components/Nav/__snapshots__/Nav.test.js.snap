// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav should match the snapshot with the data passed through 1`] = `
<nav
  className="nav"
>
  <h1>
    Deck Building
  </h1>
  <div
    className="nav-btns"
  >
    <Link
      to="/favorites"
    >
      <button
        className="nav-btn"
      >
        View Favorites
      </button>
    </Link>
    <Link
      to="/owned"
    >
      <button
        className="nav-btn"
      >
        Owned Collection
      </button>
    </Link>
    <Link
      to="/searchform"
    >
      <button
        className="nav-btn"
      >
        Search
      </button>
    </Link>
  </div>
</nav>
`;
